content/2021-08-19-GSoC21.rst

```diff
@@ -0,0 +1,101 @@
+GSoC’21 Improve performance through the use of Pythran
+######################################################
+
+:date: 2021-08-19
+:category: examples
+:lang: en
+:authors: Xingyu Liu
+:summary: This project is about using Pythran to accelerate algorithms in SciPy and 
+          writing benchmarks for the algorithms. Let's look into the details of the project.
+
+
+
+Project Overview
+================
+There are a lot of algorithms in scipy that use Cython to improve 
+the performance of code that would be too slow as pure Python, 
+e.g. algorithms in ``scipy.spatial``, ``scipy.stats`` and ``scipy.optimize``, etc. 
+Recently, SciPy added experimental support for Pythran, 
+to make it easier to accelerate Python code. 
+Compared with Cython, Pythran is more readable and even faster. 
```

Comment:
It's probably a good think to add reference to the SciPy, Cython and Pythran website for the first occurence